control_plane/attachment_service/src/http.rs

```diff
@@ -531,11 +571,100 @@ fn check_permissions(request: &Request<Body>, required_scope: Scope) -> Result<(
     })
 }
 
+#[derive(Clone, Debug)]
+struct RequestMeta {
+    method: hyper::http::Method,
+    at: Instant,
+}
+
+fn prologue_metrics_middleware<B: hyper::body::HttpBody + Send + Sync + 'static>(
+) -> Middleware<B, ApiError> {
+    Middleware::pre(move |req| async move {
+        let meta = RequestMeta {
+            method: req.method().clone(),
+            at: Instant::now(),
+        };
+        tracing::info!("Set request context meta: {meta:?}");
```

Comment:
Probably didn't mean to leave this at info level?

Reply:
Nope, definitely not: https://github.com/neondatabase/neon/pull/7178/commits/050bc0aae973eb90adc09a51d596d1a58cd1f332